docs/ad/api.md

```diff
@@ -721,14 +773,159 @@ POST _opendistro/_anomaly_detection/detectors/results/_search
+          "task_progress": 0.09486946,
+          "last_update_time": 1612126667008,
+          "execution_start_time": 1612126643455,
+          "state": "RUNNING",
+          "coordinating_node": "gs213KqjS4q7H4Bmn_ZuLA",
+          "current_piece": 1583503800000,
+          "task_type": "HISTORICAL",
+          "started_by": "admin",
+          "init_progress": 1,
+          "is_latest": true,
+          "detector": {
+            "description": "test",
+            "ui_metadata": {
+              "features": {
+                "F1": {
+                  "aggregationBy": "sum",
+                  "aggregationOf": "value",
+                  "featureType": "simple_aggs"
+                }
+              }
+            },
+            "detection_date_range": {
+              "start_time": 1580504240308,
+              "end_time": 1612126640308
+            },
+            "feature_attributes": [
+              {
+                "feature_id": "dJc8WncBgO2zoQoFWVAt",
+                "feature_enabled": true,
+                "feature_name": "F1",
+                "aggregation_query": {
+                  "f_1": {
+                    "sum": {
+                      "field": "value"
+                    }
+                  }
+                }
+              }
+            ],
+            "schema_version": 0,
+            "time_field": "timestamp",
+            "last_update_time": 1612126640448,
+            "indices": [
+              "nab_art_daily_jumpsdown"
+            ],
+            "window_delay": {
+              "period": {
+                "unit": "Minutes",
+                "interval": 1
+              }
+            },
+            "detection_interval": {
+              "period": {
+                "unit": "Minutes",
+                "interval": 10
+              }
+            },
+            "name": "test-historical-detector",
+            "filter_query": {
+              "match_all": {
+                "boost": 1
+              }
+            },
+            "shingle_size": 8,
+            "user": {
+              "backend_roles": [
+                "admin"
+              ],
+              "custom_attribute_names": [],
+              "roles": [
+                "all_access",
+                "own_index"
+              ],
+              "name": "admin",
+              "user_requested_tenant": "__user__"
+            },
+            "detector_type": "HISTORICAL_SINGLE_ENTITY"
+          },
+          "user": {
+            "backend_roles": [
+              "admin"
+            ],
+            "custom_attribute_names": [],
+            "roles": [
+              "all_access",
+              "own_index"
+            ],
+            "name": "admin",
+            "user_requested_tenant": "__user__"
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+
 ---
 
 ## Delete detector
 
 Deletes a detector based on the `detector_id`.
+To delete a historical detector, you need to first stop the detector.
```

Comment:
It's the same for realtime detector. User can't delete a running realtime detector either.